### ISSUE TITLE
Adding personal exercises

### ISSUE THREAD

**Issue by Alpenblick** [LiamMorrow/LiftLog#289]
Hi

this app would be beyond perfect if one could add personal exercises! Is this something that is planed?

Best,
Alp

**Comment by LiamMorrow** [comment]
Hi Alp,
Thanks for the feedback 😄 Can I ask what you mean by personal exercise? When you add an exercise, you can give it any name currently, you don't need to select one from the list.

**Comment by Alpenblick** [comment]
Yes but as far as I understand its not listed in the next free session? I have to enter it newly again?

**Comment by LiamMorrow** [comment]
Ah yes that's correct if you use the FreeForm session.  However if you'd like to make a plan you can like this:

https://github.com/user-attachments/assets/06d56833-a77f-45fc-afc9-18b0715166ca

Additionally, if you want to pull in some exercises from your freeform session you can go to history and select one, then update your plan from it.  Like this:

https://github.com/user-attachments/assets/a14b8981-9c64-4ce3-9ab2-f97b3fe755e6



Does that help?

**Comment by Alpenblick** [comment]
Hi Yes I think it helps, nevertheless it would be very cool if it were available automatically?

Best, Alpenblick

**Comment by LiamMorrow** [comment]
This has been completed as a part of #458 🎉
